### ISSUE TITLE
 how to read multiple log from remote host and output multiple reports

### ISSUE THREAD

**Issue by vitaocode** [allinurl/goaccess#1607]
Good afternoon, I am reading log files as follows:
`ssh -i chave.pem ubuntu@IPADDRESS 'tail -f -n +1 /var/log/apache2/other_vhosts_access.log' | goaccess -o /var/www/html/SERVER1.html -p /usr/local/etc/goaccess/goaccess.conf  --real-time-html --exclude-ip=IPADDRESS --fifo-in=/tmp/FILE.in --fifo-out=/tmp/FILE.out --port=7890`
As in the drawing:
![image](https://user-images.githubusercontent.com/31347646/69077330-25525d80-0a14-11ea-98d3-f85a3e916a83.png)

what I am trying is:
From the goaccess server, read multiple access logs.
Each access log has the log of each day, I want to analyze at least 10 days ago.
same on the preview site
like a image:
![image](https://user-images.githubusercontent.com/31347646/69077498-8417d700-0a14-11ea-8977-3674028e9028.png)

basically i want to read more than one day, but my apache rotates the logs.
I tried a preprocessing like:
ssh -i chave.pem ubuntu@IPADDRESS **'tail -f -n +1 /var/log/apache2/other_vhosts_access.log.1 & tail -f -n +1 /var/log/apache2/other_vhosts_access.log**' | goaccess -o /var/www/html/SERVER1.html -p /usr/local/etc/goaccess/goaccess.conf  --real-time-html --exclude-ip=IPADDRESS --fifo-in=/tmp/FILE.in --fifo-out=/tmp/FILE.out --port=7890

but when I try the above solution, the graphics do not display and do not make any daily access accounts or unique visitors

**Comment by allinurl** [comment]
You may get a broken pipe doing that way. What I'd do is consolidate your logs on a centralize server and use goaccess against that centralized server. Please take a look at [this](https://github.com/allinurl/goaccess/issues/532) post.

**Comment by vitaocode** [comment]
I'm not explain myself well because I'm using translator, sorry.

what I want to do is something like:
`ssh -i chave.pem ubuntu@IPADDRESS 'zcat /var/log/apache2/other_vhosts_access.log.2.gz | cat /var/log/apache2/other_vhosts_access.log.1 |tail -f -n +1 /var/log/apache2/other_vhosts_access.log' | goaccess -- options.`

i can't goaccess access.log access.log.1 because I am preprocessing the remote file via ssh,
I process the file on the remote server and read on the local server goaccess

**Comment by vitaocode** [comment]
I managed to do the way below.
![image](https://user-images.githubusercontent.com/31347646/69150162-dc9fb080-0ab6-11ea-9616-27c6a26c4aa5.png)

however, it is not dynamic when rotating the log.

![image](https://user-images.githubusercontent.com/31347646/69150224-f8a35200-0ab6-11ea-9bf3-70c18b6f52ba.png)

I want to have this vision of more days on min teen days
